batcher/batcher.mligo

```diff
@@ -145,17 +165,15 @@ let redeem (storage : storage) : result =
 
 
 let move_current_to_previous_if_finalized (storage : storage) : storage =
-  let batches = storage.batches in
-  let current = batches.current in
+  let batch_set = storage.batch_set in
```

Comment:
I think we need to change the name of this function to `move_current_to_last_if_finalized`.

Reply:
I am not sure that make much of a difference